### PR TITLE
Add some typescript options for angular 2

### PR DESCRIPTION
**Pull request by wassertim**
This should close the issue: https://github.com/ArpNetworking/sbt-typescript/issues/23

@hseeberger please review and test :) 
@BrandonArp could you merge it if everything is alright?

### CODE REVIEW

File: src/main/resources/typescriptc.js [hunk 1]
```diff
@@ -19,7 +19,10 @@
 (function () {
 
     "use strict";
-
+    var moduleResolutionKind = {
```
Comment (BrandonArp) [inline src/main/resources/typescriptc.js]: Is there a typescript compiler map that already has these?

**Comment by BrandonArp** [comment]
Can you add some tests that show that these work?

File: src/main/resources/typescriptc.js (hunk 1 above)
Comment (wassertim) [inline src/main/resources/typescriptc.js]: There is a type definition for this, I will check if there is a map also.

**Comment by wassertim** [comment]
Yes, sure I will.

**Comment by wassertim** [comment]
Hey @BrandonArp. I made the changes we agreed on:
- A test for angular2 options
- Mappings for the options

It also has some breaking changes (so maybe to consider to make a major release), because of the typescript option mapping. 

I also edited the options comments accordingly to the changes.

Please review. I'm looking forward to merging this ;)

File: src/sbt-test/sbt-typescript-plugin/angular2-options/project/build.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+sbt.version=0.13.5
```
Comment (bjoernhaeuser) [inline src/sbt-test/sbt-typescript-plugin/angular2-options/project/build.properties:1]: Suggesting to use the newest version here

Comment (wassertim) [inline src/sbt-test/sbt-typescript-plugin/angular2-options/project/build.properties:1]: It seems it breaks the continuous integration :(

Comment (bjoernhaeuser) [inline src/sbt-test/sbt-typescript-plugin/angular2-options/project/build.properties:1]: Would also suggest to upgrade the jdk to 8 and scala to 2.11 (in .travis.yml)

**Comment by BrandonArp** [comment]
Thanks for all the work here.  I agree that we need to bump the major version.  Please downgrade the sbt version so that CI passes and then I'll merge.  If you want, you can also add a Scala 2.11 target to the .travis.yml file.  

Sorry I didn't get this merged yesterday.  I had time set aside, but github was down when I was available.

**Comment by wassertim** [comment]
Great :) When do you release a new version?

**Comment by BrandonArp** [comment]
Just promoted on Sonatype.  Version is 0.2.0 and it should be in central in ~10 minutes.

**Pull request merged**